src/forms/output/Summary.js

```diff
@@ -342,6 +342,8 @@ const Summary = function ({ client, openFeedback, snippets }) {
   }  // ends for each benefit
 
   // Ask for feedback
+  // Stays put when printing. A take-home hint
+  // that the tool is still a prototype
```

Comment:
Hm, I'm not seeing this in the print preview. Not sure why.

Reply:
Weird, I'll double check later today.

Reply:
I did it now instead, and I do see the 'let us know' line when there's a cliff. The button just doesn't have a background.

Reply:
If you don't see it, could you post an image here of what you do see?

Reply:
Oh, I didn't realize it was only supposed to show up if there's a cliff?

Reply:
Oh never mind, I wasn't reading closely and I thought this was the yellow bar asking for feedback above the tabs, for some reason.